database/build.js

```diff
@@ -56,4 +56,11 @@ const writeStream = (writer, len) => {
   write();
 };
 
-writeStream(stream, 20000);
\ No newline at end of file
+writeStream(stream, 20000);
+
+module.exports = {
+  paragraphs: paragraphs,
+  randomize: randomize,
+  createListing: createListing,
+  roomType: roomType
+};
```

Comment:
Some minor things, remember to include an extra line at the end of the file. Also, since your keys are the same as the value variables you can do {paragraphs, randomize, createListing, roomType}